src/app/models/question-dto.ts

```diff
@@ -0,0 +1,36 @@
+export class QuestionDto {
+    public readonly title: string;
+    public readonly body: string;
+    public readonly sampleAnswer: string;
+    public readonly hints: string[];
+    public readonly successRate: number;
+    public readonly difficulty: string;
+    public solved: boolean;
+
```

Comment:
for consistency question objects in the backend will be made to look like this on the next commit.